report/renderer/report-utils.js

```diff
@@ -91,7 +105,15 @@ class ReportUtils {
         }
 
         // Attach table/opportunity items with entity information.
-        ReportUtils.classifyEntities(result.entities, audit);
+        if (audit.details.type === 'opportunity' || audit.details.type === 'table') {
+          // third-party-summary produces a per-entity aggregated result.
+          if (audit.id === 'third-party-summary') {
+            audit.details.isAggregated = true;
```

Comment:
We could output this directly from the audit as well, at the cost of extra LHR payload.